CONTRIBUTING.md

```diff
@@ -24,3 +24,12 @@ of this plugin:
 - Create the release on GitHub for that tag
     - Build the release package with `./ci/build_release`
     - Attach package to Github Release
+
+### Testing
+
+To run the tests:
+
+```sh-session
+$ cd tests
+$ ./test.sh <conjur or conjur-host-identity>
```

Comment:
could we make this more clear? could we name the flags for the functions, or something like `lookup_plugin` and `role`?